### PR TITLE
Add subject list in requirement form

### PR DESCRIPTION
**Pull request by dimasciput**
For #78 

ford

### CODE REVIEW

**Comment by pierrealixt** [comment]
@dimasciput see 
https://github.com/kartoza/ford3/issues/91

**Comment by dimasciput** [comment]
> @dimasciput see
> #91

@pierrealixt we can do that in another PR, because that's just populating database

File: django_project/ford3/models/subject.py [hunk 1]
```diff
@@ -6,24 +6,16 @@ class Subject(models.Model):
...
-        unique=False,
         help_text='',
         max_length=255)
     description = models.CharField(
         blank=True,
         null=True,
-        unique=False,
         help_text='',
-        max_length=255)
+        max_length=500)
```
Comment (lucernae) [inline django_project/ford3/models/subject.py:18]: Maybe change this to TextField if the text is going to be very very long

File: django_project/ford3/views/qualification_wizard.py [hunk 2]
```diff
@@ -85,12 +87,35 @@ def validate_form_data(self, form_data):
             **qualification_fields
         )
 
+        # Subjects
+        post_dict = self.request.POST
+        subject_length = post_dict.get('subject-length', 1)
+        for subject_index in range(int(subject_length)):
+            subject = None
+            subject_index += 1
```
Comment (lucernae) [inline django_project/ford3/views/qualification_wizard.py]: I don't understand why you manipulate the index?
Does it help if the range starts from 1?

File: django_project/ford3/views/qualification_wizard.py [hunk 3]
```diff
@@ -85,12 +87,35 @@ def validate_form_data(self, form_data):
...
+            subject_key = '2-subject'
+            minimum_score_key = 'subject-minimum-score'
+            if subject_index > 1:
+                subject_key += '_%s' % subject_index
```
Comment (lucernae) [inline django_project/ford3/views/qualification_wizard.py]: Let's use python 3 style formatting :)

**Review by lucernae — APPROVED** [review]
LGTM with some comments.
I think we should add unittests to check the form from this point

**Pull request merged**